ui/src/app/applications/components/application-status-panel/application-status-panel.tsx

```diff
@@ -24,20 +24,20 @@ interface SectionInfo {
 }
 
 const sectionLabel = (info: SectionInfo) => (
-    <label>
+    <label style={{fontSize: '12px', fontWeight: 600, color: ARGO_GRAY6_COLOR}}>
         {info.title}
         {info.helpContent && <HelpIcon title={info.helpContent} />}
     </label>
 );
 
 const sectionHeader = (info: SectionInfo, hasMultipleSources: boolean, onClick?: () => any) => {
     return (
-        <div style={{display: 'flex', alignItems: 'center'}}>
+        <div style={{display: 'flex', alignItems: 'center', marginBottom: '0.5em'}}>
             {sectionLabel(info)}
             {onClick && (
-                <button className='argo-button argo-button--base-o argo-button--sm application-status-panel__more-button' onClick={onClick} disabled={hasMultipleSources}>
+                <button className='application-status-panel__more-button' onClick={onClick} disabled={hasMultipleSources}>
                     {hasMultipleSources && helpTip('More details are not supported for apps with multiple sources')}
-                    &nbsp; MORE
+                    <i className='fa fa-ellipsis-v' />
```

Comment:
Does this imply a drop down menu to appear like in the other places where we use the three vertical dots (eg. app list view)?  Although it is specified in the design, it doesn't seem consistent in behavior.

Reply:
Good point. I think a horizontal ellipsis would stay close to the original design but be less confusing. I'll change this

Reply:
<img width="355" alt="CleanShot 2023-02-08 at 09 18 12@2x" src="https://user-images.githubusercontent.com/6250584/217604115-008ce5c1-019b-43a3-b5da-6b4e0132aaee.png">